Sources/Workspace/Workspace.swift

```diff
@@ -499,14 +496,6 @@ extension Workspace {
         let graphRoot = PackageGraphRoot(input: root, manifests: rootManifests)
         let currentManifests = loadDependencyManifests(root: graphRoot, diagnostics: diagnostics)
 
-        // Abort if we're unable to load the pinsStore or have any diagnostics.
-        guard let pinsStore = diagnostics.wrap({ try self.pinsStore.load() }) else {
-            return
-        }
-
-        // Ensure we don't have any error at this point.
-        guard !diagnostics.hasErrors else { return }
```

Comment:
We should keep this check because some of the other operations might fail

Reply:
done